packages/build-info/src/frameworks/waku.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseFramework, Category, Framework } from './framework.js'
+
+export class Waku extends BaseFramework implements Framework {
+  readonly id = 'waku'
+  name = 'Waku'
+  npmDependencies = ['waku']
+  configFiles = ['waku.config.ts', 'waku.config.ts']
```

Comment:
typo?
```suggestion
  configFiles = ['waku.config.ts', 'waku.config.js']
```

Source: https://github.com/wakujs/waku/blob/27dcf0b2d0562580b47a421feb500952903625c5/packages/waku/src/lib/vite-rsc/cli.ts#L12

Reply:
yeah typo... sorry 🤦